debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/commands/J9ConstantPoolCommand.java

```diff
@@ -0,0 +1,507 @@
+						}
+					} else {
+						if ((i < ramCPCount) && clazz.isNull()) {
+							ramInfo = "(unresolved)";
+						}
+						J9ClassPointer foundClass = findClassByName(vm, className);
+						if (foundClass == null) {
+							romInfo = "(unresolved)";
+						} else if (J9ClassHelper.isArrayClass(foundClass)) {
+							romInfo = "!j9romarrayclass " + foundClass.romClass().getHexAddress();
+						} else {
+							romInfo = "!j9romclass " + foundClass.romClass().getHexAddress();
+						}
+					}
+					value = className;
+				} else if (cpType == J9ConstantPool.J9CPTYPE_STRING) {
+					romInfo = "!j9utf8 " + J9ROMStringRefPointer.cast(romCPP).utf8Data().getHexAddress();
+					if (i < ramCPCount) {
+						J9ObjectPointer stringObject = J9RAMStringRefPointer.cast(ramCPP).stringObject();
+						if (stringObject.notNull()) {
+							ramInfo = "!j9object " + stringObject.getHexAddress();
+						} else {
+							ramInfo = "(unresolved)";
+						}
+					}
+					value = J9UTF8Helper.stringValue(J9ROMStringRefPointer.cast(romCPP).utf8Data());
+				} else if (cpType == J9ConstantPool.J9CPTYPE_INT) {
+					J9ROMSingleSlotConstantRefPointer constantRef = J9ROMSingleSlotConstantRefPointer.cast(romCPP);
+					romInfo = "!J9ROMSingleSlotConstantRef " + constantRef.dataEA().getHexAddress();
+					if (i < ramCPCount) {
+						ramInfo = "!J9RAMSingleSlotConstantRef " + ramCPP.getHexAddress();
+					}
+					value = Long.toString(constantRef.data().longValue());
+				} else if (cpType == J9ConstantPool.J9CPTYPE_FLOAT) {
+					J9ROMSingleSlotConstantRefPointer constantRef = J9ROMSingleSlotConstantRefPointer.cast(romCPP);
+					romInfo = "!J9ROMSingleSlotConstantRef " + constantRef.getHexAddress();
+					if (i < ramCPCount) {
+						ramInfo = "!J9RAMSingleSlotConstantRef " + ramCPP.getHexAddress();
+					}
+					value = Float.toString(Float.intBitsToFloat((int)constantRef.data().longValue()));
+				} else if (cpType == J9ConstantPool.J9CPTYPE_LONG) {
+					// long and double don't have an entry in RAM CP
+					romInfo = "!j9romconstantref " + romCPP.getHexAddress();
+					value = Long.toString(I64Pointer.cast(romCPP).at(0).longValue());
+				} else if (cpType == J9ConstantPool.J9CPTYPE_DOUBLE) {
+					romInfo = "!j9romconstantref " + romCPP.getHexAddress();
+					value = Double.toString(Double.longBitsToDouble(I64Pointer.cast(romCPP).at(0).longValue()));
+				} else if (cpType == J9ConstantPool.J9CPTYPE_FIELD) {
+					if (i < ramCPCount) {
+						J9RAMStaticFieldRefPointer staticRef = J9RAMStaticFieldRefPointer.cast(ramCPP);
+						J9RAMFieldRefPointer ref = J9RAMFieldRefPointer.cast(ramCPP);
+						if ((staticRef.flagsAndClass().longValue() > 0) && !staticRef.valueOffset().eq(UDATA.MAX)) {
+							ramInfo = staticRef.valueOffset().getHexValue() + " (Static offset)"; // resolved static
+						} else if (ref.flags().gt(ref.valueOffset())) {
+							ramInfo = ref.valueOffset().getHexValue() + " (Instance offset)"; // resolved instance
```

Comment:
Suggest "static" and "instance" (with a lowercase first character like the Java keywords).